ci-operator/step-registry/openshift/e2e/gcp/libvirt/conf/openshift-e2e-gcp-libvirt-conf-ref.yaml

```diff
@@ -8,11 +8,11 @@ ref:
       memory: 100Mi
   env:
     - name: HOME
-      default: /tmp/packer
+      default: /tmp/secret
```

Comment:
would like to use `${SHARED_DIR}` but not sure how to make that substitution work. I've tried several 
${SHARED_DIR}, "${SHARED_DIR}", '${SHARED_DIR}'

Reply:
```
NWRAP_ERROR(119) - nwrap_files_cache_reload: Unable to open '${SHARED_DIR}/passwd' readonly -1:No such file or directory
NWRAP_ERROR(119) - nwrap_files_getpwuid: Error loading passwd file
NWRAP_ERROR(123) - nwrap_files_cache_reload: Unable to open '${SHARED_DIR}/passwd' readonly -1:No such file or directory
NWRAP_ERROR(123) - nwrap_files_getpwuid: Error loading passwd file
```